src/client/components/ChannelHeadNav/ChannelHeadNav.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import './ChannelHeadNav.style.css';
+
+export default function ChannelHeadNav({ channelName, urlBack, imgUrl }) {
+  const backArrow = '<';
+  return (
+    <div className="channel-head-div">
+      <Link to={urlBack} className="channel-head-back-arrow-icon">
+        <h1>{backArrow}</h1>
```

Comment:
There should be only 1 h1 on a page and that is the main header of the page. This could affect SEO.